dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -47,6 +89,29 @@ def refute_estimate(self, *args, **kwargs):
                                     n_samples=self._sample_size,
                                     random_state=self._random_state )
 
+            if self._chosen_variables is not None:
+                for variable in self._chosen_variables:
+                    
+                    if ('float' or 'int') in new_data[variable].dtype.name: 
+                        new_data[variable] += np.random.randn(self._sample_size) * self._noise

```

Comment:
this is an odd construction. why not sample a normal variable with 0 mean and std dev=noise?

Reply:
Also, how do we account for the fact that different variables may have different magnitude? A noise of 1 on a variable that ranges in the thousands may have no effect.

Probably we should parameterize noise by the standard deviation of each feature, perhaps as a fraction of it? That has a nice property that both noise parameters for continuous and binary features are usually between [0,1]